classes/AnyPage.php

```diff
@@ -32,8 +44,8 @@ public function save() {
 		$site = elgg_get_site_entity();
 
 		$this->access_id = ACCESS_PUBLIC;
-		$this->container_guid = $site->getGUID();
-		$this->owner_guid = $site->getGUID();
+		$this->container_guid = $site->guid;
+		$this->owner_guid = $site->guid;
```

Comment:
Where? It should never be too late to improve practices, and definitely not for other projects.